app.go

```diff
@@ -151,59 +152,73 @@ func (a *App) GetTemplates() []exporterAPI.TemplateResponseItem {
 }
 
 // CheckApiKey validates the api key from the config file if it exists
-func (a *App) CheckApiKey() bool {
+func (a *App) CheckApiKey() string {
 	token := a.cm.Configuration.AccessToken
 	if len(token) == 0 {
-		return false
+		return "token can't be blank"
```

Comment:
If this is customer facing. We should use something more like "Please enter an API key"